includes/test-user.php

```diff
@@ -89,6 +98,8 @@ public function handleUserAction()
                     $redirect_url = admin_url();
                 }
 
+                $original_user_id = $current_user_id;
```

Comment:
@olatechpro The new hooks look fine, but here's a very minor critique. Why set $original_user_id = $current_user_id here rather than just passing $current_user_id into the subsequent do_action() ?  Is this just for clarity in matching the doc block action argument name?

Reply:
Yes, my reason is for both clarity and consistency with the the 2 action hook. I also feel like having consistent $original_user_id parameter in both hook should make it easier to know it's the main user that's performing the switch or switch back action compared to having $current_user_id which seems to be straight forward as currently logged in user but could be confusing on if it's for the logged in user before switch or logged in user after switch back. @agapetry